.github/workflows/test.yml

```diff
@@ -0,0 +1,30 @@
+on: [push, pull_request]
+name: Test
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-version: [1.14.x, 1.15.x]
+        os: [ubuntu-latest, macos-latest, windows-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{ matrix.go-version }}
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: Test
+      run: |
+        go test ./...
+        go test -race ./...
+
+    - name: Tidy
+      if: matrix.os == 'ubuntu-latest' # no need to do this everywhere
+      run: |
+        go mod tidy
+        # https://github.com/golang/go/issues/27868#issuecomment-431413621
+        go list all > /dev/null
```

Comment:
This can be dropped now, was fixed back in 1.12/1.13 from memory.

Reply:
ack, will do